about/about-networking.md

```diff
@@ -0,0 +1,152 @@
+url='https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing' %} that consists of an IP address and number of
+significant bits on the IP address separated by a `/`. For example, `192.168.27.0/24` represents the group of 256 IP
+addresses from `192.168.27.0` to `192.168.27.255`.
+
+A group of IP addresses within a single L2 network is referred to as a subnet. Within a subnet, packets can be sent
+between any pair of devices as a single network hop, based solely on the L2 header (and footer).
+
+To send packets beyond a single subnet requires L3 routing, with each L3 network device (router) being responsible for
+making decisions on the path to send the packet based on L3 routing rules. Each network device acting as a router has
+routes that determine where a packet for a particular CIDR should be sent next. So for example, in a Linux system, a
+route of `10.48.0.128/26 via 10.0.0.12 dev eth0` indicates that packets with destination IP address in `10.48.0.128/26`
+should be routed to a next network hop of `10.0.0.12` over the `eth0` interface.
+
+Routes can be configured statically by an administrator, or programmed dynamically using routing protocols. When using
+routing protocols each network device typically needs to be configured to tell it which other network devices it should
+be exchanging routes with. The routing protocol then handles programming the right routes across the whole of the
+network as devices are added or removed, or network links come in or out of service.
+
+One common routing protocol used in large enterprise and data center networks is {% include open-new-window.html
+text='BGP' url='https://en.wikipedia.org/wiki/Border_Gateway_Protocol' %}. BGP is one of the main protocols that powers
+the internet, so scales incredibly well, and is very widely supported by modern routers.
+
+### Overlay networks
+
+An overlay network allows network devices to communicate across an underlying network (referred to as the underlay)
+without the underlay network having any knowledge of the devices connected to the overlay network. From the point of
+view of the devices connected to the overlay network, it looks just like a normal network. There are many different
+kinds of overlay networks that use different protocols to make this happen, but in general they share the same common
+characteristic of taking a network packet, referred to as the inner packet, and encapsulating it inside an outer network
+packet. In this way the underlay sees the outer packets without needing to understand how to handle the inner packets.
+
+How the overlay knows where to send packets varies by overlay type and the protocols they use. Similarly exactly how the
+packet is wrapped varies between different overlay types.  In the case of VXLAN for example, the inner packet is wrapped
+and sent as UDP in the outer packet.
+
+![Anatomy of an overlay network packet]({{site.baseurl}}/images/anatomy-of-an-overlay-packet.svg)
+
+### DNS
```

Comment:
Should this be at the top?  It's the first layer (that we discuss) that gets hit from an application developers perspective.